### PR TITLE
Use Zap logger

### PR DESCRIPTION
**Pull request by levb**
uze zap logger, consistently

- added `utils.Logger` - a thin wrapper on zap that uses p.mm.Log
- added `p.log` the main instance, and pass it around to all `services`
- Use `.Debugf` for unstructured, `Debugw` for structured

### CODE REVIEW

File: server/httpin/dialog/install.go [hunk 1]
```diff
@@ -176,7 +176,7 @@ func (d *dialog) handleInstall(w http.ResponseWriter, req *http.Request) {
 
 	_, out, err := d.proxy.InstallApp(client, sessionID, cc, noUserConsentForOAuth2, secret, "")
 	if err != nil {
-		d.mm.Log.Warn("Failed to install app", "app_id", cc.AppID, "error", err.Error())
+		d.log.WithError(err).Warnw("Failed to install app", "app_id")
```
Comment (larkox) [inline server/httpin/dialog/install.go:179]: Are you missing here `cc.AppID`? Or is it being passed somehow in a different way?

**Review by larkox — APPROVED** [review]
LGTM

File: server/httpin/dialog/install.go (hunk 1 above)
Comment (levb) [reply server/httpin/dialog/install.go:179]: TY for the catch

**Review by mickmister — APPROVED** [review]

**Comment by levb** [comment]
@DHaussermann I am going to skip QA review on this?

**Pull request merged**